app/components/channel-container/template.hbs

```diff
@@ -9,16 +9,22 @@
       {{this.channel.formattedTopic}}
     </p>
     <nav>
-      <button type="button"
-              {{action "menu" "channel" "toggle"}}
-              class={{if this.headerNavButtonUsersActive "active"}}>
+      <LinkTo @route="space.channel.index" @models={{array space channel}} {{action "menu" "channel" "show"}}>
```

Comment:
Shouldn't this rather be `[...] {{array this.space this.channel}} [...]`?

Reply:
Yes, there are hundreds of Octane warnings still. Everyone is invited to help with the upgrade.